sqlmesh/core/model/definition.py

```diff
@@ -1606,6 +1612,11 @@ def violated_rules_for_query(self) -> t.Dict[type[Rule], t.Any]:
         self.render_query()
         return self._query_renderer._violated_rules
 
+    @property
+    def _raw_code(self) -> t.Optional[str]:
+        query = self.query
+        return query.name if isinstance(query, d.JinjaQuery) else None
```

Comment:
btw, you can now do `model.query_.sql` to access the original raw sql of any model

Reply:
ah, no, nvm. I think case it will have `JINJA_*` block which is undesirable